css_styles/styles.css

```diff
@@ -134,9 +142,20 @@ div ul li {
 }
 
 /* works section styles */
+#myworks {
+  display: grid;
+  width: 100%;
+  height: auto;
+  background-color: #e5e5e5;
```

Comment:
Kindly remove this background color to make your section look like figma design check the template again, please do all changes requested by the previous reviewers as this was mentioned before 

![temp2](https://user-images.githubusercontent.com/59707859/236624121-bb94b1d3-e663-4c7d-9076-3c50bd48bb11.PNG)

Reply:
I have done the changes as you mentioned and I am very thankful to you for highlighting this.